seed/static/seed/js/controllers/inventory_list_controller.js

```diff
@@ -1013,6 +1013,7 @@ angular.module('BE.seed.controller.inventory_list', [])
         $uibModal.open({
           templateUrl: urls.static_url + 'seed/partials/export_inventory_modal.html',
           controller: 'export_inventory_modal_controller',
+          backdrop: 'static',
```

Comment:
This prevents the modal from closing if the user clicks outside of the modal. I found accidentally closing the modal and losing the visual progress to be annoying. Open to reverting to the original functionality.